### DISCUSSION TITLE
有问题了

### DISCUSSION THREAD

**Discussion by 15941127712ly** [cotaku/Bibi-Release#183] [Q&A]
![image](https://github.com/user-attachments/assets/a526bf52-07cb-4043-8438-40d4db2119bb)
大佬，视频刷不出来了

**Comment by 15941127712ly** [comment]
还有分区栏目
也刷不出来了

**Comment by cotaku** [comment]
/dup #165
在修了在修了

**Reply by 15941127712ly** [reply]
谢谢大佬
